app/views/miq_ae_customization/_dialog_field_form.html.haml

```diff
@@ -59,10 +59,16 @@
   - unless @edit[:field_typ].nil?
     %h3
       = _('Options')
-
     .form-horizontal
       .form-group
         %label.col-md-2.control-label
+          = _('Visible')
+        .col-md-8
+          = check_box_tag('field_visible', '1',
+                          @edit[:field_visible],
+                          'data-miq_observe_checkbox' => {:url => url}.to_json)
+      .form-group
+        %label.col-md-2.control-label
```

Comment:
With #9423 merged, shouldn't this block go into the non_dynamic_options partial?